src/meta-srv/Cargo.toml

```diff
@@ -6,13 +6,15 @@ license.workspace = true
 
 [features]
 mock = []
+pg_kvbackend = ["dep:tokio-postgres"]
 
 [lints]
 workspace = true
 
 [dependencies]
 api.workspace = true
 async-trait = "0.1"
+clap.workspace = true
```

Comment:
Sorry, Why we need this?

Reply:
we need this to derive `ValueEnum`  for `BackendImpl` enum